tests/admin/valuesets/valueset-creation.spec.ts

```diff
@@ -0,0 +1,380 @@
+import { expect, test } from "@playwright/test";
+
+// Use the authenticated state
+test.use({ storageState: "tests/.auth/user.json" });
+
+test.describe("ValueSet Creation", () => {
+  test.beforeEach(async ({ page }) => {
+    // Navigate to ValueSets management
+    await page.goto("/");
+    await page.waitForLoadState("networkidle");
+    await page.getByRole("link", { name: "Admin Dashboard" }).click();
+    await page.waitForLoadState("networkidle");
+
+    // Navigate to ValueSets page
+    await page.getByRole("button", { name: "Toggle Sidebar" }).click();
+    await page.getByRole("link", { name: "ValueSets" }).click();
+    await page.waitForLoadState("networkidle");
+
+    // Start creating new ValueSet
+    await page.getByRole("button", { name: "+ Create ValueSets" }).click();
+    await page.waitForLoadState("networkidle");
+  });
+
+  test("should create a valueset with just required fields", async ({
```

Comment:
- there should similar test for 3 system, then only before creating you can open the preview tab and verify, the expected dropdown are only visible